Snakefile

```diff
@@ -780,7 +780,12 @@ rule trinity_stats:
     1
   shell:
     """
-    {TRINITY_HOME}/util/TrinityStats.pl {input.transcriptome} > {output.stats} 2> {log}
+    assembler={config[assembler]}
+    if [ "$assembler" = 'trinity' ]; then
```

Comment:
I think this should be `==`

Reply:
I think in bash `=` and `==` are equivalent

Reply:
Ok